src/View/Helper/PaginationHelper.php

```diff
@@ -144,13 +144,13 @@ public function display($extramParams = array(), $extraNumbersOptions = array())
      *
      * @return string
      **/
-    public function sortForRole() {
+    public function sortForRole($label = 'Member', $options = array()) {
```

Comment:
It looks like this function is not needed any more, so you might as well remove it altogether.

Reply:
I was not completely sure, but now I removed it.